estate_account/models/estate_property.py

```diff
@@ -0,0 +1,34 @@
+from odoo import fields, models, Command
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def sell_property(self):
+        res = super().sell_property()
+
+        for record in self:
+            self.env["account.move"].create(
+                {
+                    "partner_id": record.buyer_id.id,
```

Comment:
to avoid too much indentation :
```
self.env["account.move"].create({
    "partner_id"
})
```
But, create should be multi, so dict value should be inside a list.
```
self.env["account.move"].create([{
    "partner_id"
}, {
  # dict 2
}])
```
And, do call search / create etc.. in loop.
build a list of dict values to create, and create everything at the end
```
create_vals = [{
    "partner_id": property.buyer_id.id,
     ...
} for property in self]
self.env["account.move"].create(create_vals)
```